src/components/study-pane.js

```diff
@@ -79,7 +79,9 @@ const StudyPane = () => {
 
     const [studyNotFound, setStudyNotFound] = useState(false);
 
-    const [voltageLevelId, setVoltageLevelId] = useState(null);
+    const [displayedVoltageLevelId, setDisplayedVoltageLevelId] = useState(null);
+
+    const [focusedVoltageLevelId, setFocusedVoltageLevelId] = useState(null);
```

Comment:
What is the purpose of this variable?

Reply:
"voltageLeveld" was initially used to open the diagram from the clicked voltage level in the explorer. Now I have also to share an another ID for the map from an action in the explorer. That's why there are now two variables in the study-pane common context. Do you have a suggestion to share this information ?

Reply:
ok I understand